### ISSUE TITLE
tke-aplication should support healthcheck

### ISSUE THREAD

**Issue by leoryu** [tkestack/tke#1396]
<img width="343" alt="企业微信截图_b637cb6c-c052-4614-82e5-74808212af7e" src="https://user-images.githubusercontent.com/34183243/124211080-1fdd5480-db1f-11eb-8ff3-77c08685aac5.png">

`deployed` can't reflect the current status of application.

**Comment by xiaosuiba** [comment]
Yes, this bothered me a lot, and I guess the way kubeapps do could be a good example.
